tdrs-backend/tdpservice/users/test/test_auth_check.py

```diff
@@ -0,0 +1,34 @@
+"""Test the authorization check."""
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+
+@pytest.mark.django_db
+def test_auth_check_endpoint_with_no_user(api_client, user):
+    """If there is no user auth_check should return FORBIDDEN."""
+    response = api_client.get(reverse("authorization-check"))
```

Comment:
that `reverse()` util is nifty!

Reply:
Right? It allows us the flexibility to change endpoints in the future should it be needed without having to update the code